resources/stakeholder_workshops/user_research_questions.md

```diff
@@ -0,0 +1,53 @@
+# Process & Guidance for Designing Effective User Research Questions
+
+## Pre-work before drafting usability questions
+
+- With team, specify features of MVP
+- With team, specify Experiment / Hypotheses around current MVP
+- With team draft user journeys
```

Comment:
- With team, identify target users (ideally specific teams at organisations, otherwise generate user personas)

Reply:
(seems like a missing piece before conducing user research and defining user journeys is clearly defining who are the users/early adopters?)